__tests__/apps.test.js

```diff
@@ -320,3 +320,62 @@ describe("6.  GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("7 POST /api/articles/:article_id/comments", () => {
+  test("status:201, responds with the newly posted comments", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "I hope its worth the hype",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
```

Comment:
can compare author and body to the strings in your newComment variable, to ensure it returns the newly posted comment